devicetypes/smartthings/zigbee-button.src/zigbee-button.groovy

```diff
@@ -64,7 +65,20 @@ def parse(String description) {
     log.debug "description is $description"
     def event = zigbee.getEvent(description)
     if (event) {
-        sendEvent(event)
+        if ((device.getDataValue("model") == "SOS-EM")) {
+            if (description?.startsWith('zone status')) {
```

Comment:
This code won't be hit as this is in the branch where `getEvent` returned an event, and neither `zone status` messages nor battery are handled in the generic handling.